.travis.yml

```diff
@@ -16,5 +16,7 @@ matrix:
       env: TOXENV=flake8
     - python: 2.7
       env: TOXENV=ansible-lint
+    - python: 2.7
+      env: TOXENV=ansible2-lint
```

Comment:
So I'd caution against calling it templated. This isn't templates so much as it is substitution.

Also if the deps do not change between testenvs you don't need to specify them, e.g.,

```ini
[testenv]
setenv=
    ANSIBLE_VERSION=""
basepython = python2.7
deps =
    -rtest-requirements.txt
    ansible{env:ANSIBLE_VERSION}

[testenv:ansible-lint]
setenv=
    ANSIBLE_VERSION=>=1.9.1,<2.0.0
commands =
    {toxinidir}/scripts/linting-ansible.sh

[testenv:ansible2-lint]
setenv=
    ANSIBLE_VERSION=>=2.0
commands = {[testenv:ansible-lint]commands}
```